Source/WebKitLegacy/mac/Scripts/PreferencesTemplates/WebPreferencesInternalFeatures.mm.erb

```diff
@@ -43,7 +43,7 @@ using namespace WebKit;
 <%- if @pref.condition -%>
 #if <%= @pref.condition %>
 <%- end -%>
-        adoptNS([[WebFeature alloc] initWithKey:@"<%= @pref.name %>" preferenceKey:@"<%= @pref.preferenceKey %>" name:@<%= @pref.humanReadableName %> status:<%= @pref.webFeatureStatus %> details:@<%= @pref.humanReadableDescription %> defaultValue:DEFAULT_VALUE_FOR_<%= @pref.name %> hidden:<%= @pref.hidden %>]).get(),
+        adoptNS([[WebFeature alloc] initWithKey:@"<%= @pref.name %>" preferenceKey:@"<%= @pref.preferenceKey %>" name:@<%= @pref.humanReadableName %> status:<%= @pref.webFeatureStatus %> category:WebFeatureCategoryNone details:@<%= @pref.humanReadableDescription %> defaultValue:DEFAULT_VALUE_FOR_<%= @pref.name %> hidden:<%= @pref.hidden %>]).get(),
```

Comment:
Why not pass the correct category in here?

Reply:
Good point -- I think after your suggested generator change there is never a case where this is unknown.